Kernel/Process.h

```diff
@@ -248,11 +253,11 @@ class Process final
 
     // Breakable iteration functions
     template<IteratorFunction<Process&> Callback>
-    static void for_each(Callback);
-    template<IteratorFunction<Process&> Callback>
-    static void for_each_in_pgrp(ProcessGroupID, Callback);
-    template<IteratorFunction<Process&> Callback>
-    void for_each_child(Callback);
+    static void for_globally_each(Callback);
```

Comment:
I think naming this `for_each_globally` would read better and match the naming pattern of the other for_each_* methods below.